### DISCUSSION TITLE
TP6 - Los ejercicios ¿van todos juntos o separados?

### DISCUSSION THREAD

**Discussion by martinjguevara** [INGCOM-UNRN-P1/cursada-2023#178] [Preguntas y respuestas] [answered]
Los primeros dos ejercicios del TP deben estar en archivos .c separados? o los dos en un mismo archivo .c? Me genero la duda que el ejemplo del ejercicio "Ordena" hace uso de la función esta_ordenado

**Comment by martinvilu** [comment, marked as answer]
Siendo que cada ejercicio debe ser un programa separado, cada uno con sus propios argumentos. Sí.

Es _técnicamente_ posible dejar todo junto, pero los scripts que luego van a hacer uso de esos programas van a requerir modificaciones.
